pages/content/amp-dev/documentation/courses/index.html

```diff
@@ -1,13 +1,13 @@
 ---
-$title: Learn AMP with free courses
```

Comment:
I actually liked that this showed what kind of resource the user can expect over other potential SERP items 🙂Were there specific reasons going for just "Learn AMP"?

Reply:
Yes - I thought the previous title looked too long to me in the sidebar - too much like an ad. I hadn't realized that this is what also appears in the SERP. Of course, it is. Don't suppose there's a way to get one thing in the sidebar and another in the `<title>` tag?